prime-router/src/main/kotlin/fhirengine/translation/hl7/ConfigSchemaProcessor.kt

```diff
@@ -109,11 +109,12 @@ abstract class ConfigSchemaProcessor : Logging {
         element: ConfigSchemaElement,
```

Comment:
We probably ought to at least mention `contextResource` in the function comment, though there's not much detail anyway, so this is more of a nit.